### DISCUSSION TITLE
Labels in discussions? [meta]

### DISCUSSION THREAD

**Discussion by JoshuaKGoldberg** [renovatebot/renovate#24449] [Suggest an Idea]
### Tell us more.

I just tried to search for docs issues in these GitHub Discussions before filing #24448. None of the discussions are labeled with `type:docs`. In fact, out of ~29 pages of discussions, ~26 do not have a label.

Would it be possible for the owning team(s) to label discussions here for discoverability please? Or otherwise provide some easier way to categorize and/or search through existing discussions?

**Comment by rarkins** [comment]
No, discussions are just that. Anything which becomes an issue then get labels
